src/pages/appointment/participant/[appointment_id].tsx

```diff
@@ -0,0 +1,201 @@
+          <Card sx={ParticipantAppointmentStyle.Card}>
+            <Stack spacing={0} alignItems="center" justifyContent="center">
+              {/* Post title */}
+              <Box style={ParticipantAppointmentStyle.TextField}>
+                <TitleTextField
+                  name="title"
+                  header="Title"
+                  placeholder="This is Post Title"
+                  value={appointment.detailHeader.title}
+                  handleValueChange={() => {}}
+                  isErr={false}
+                  errMsg=""
+                  readOnly={true}
+                />
+              </Box>
+
+              {/* Location */}
+              <Box sx={ParticipantAppointmentStyle.TextField}>
+                <LocationTextField
+                  header="Location"
+                  placeholder="Enter Location"
+                  initialValue={appointment.detailHeader.location}
+                  onChange={() => {}}
+                  isErr={false}
+                  errMsg=""
+                  readOnly={true}
+                />
+              </Box>
+
+              {/* Date & Time */}
+              <Box sx={ParticipantAppointmentStyle.TextField}>
+                <DisplayDateTime
+                  header="Date & Time"
+                  value={`${appointment.detailHeader.startDateTime} - ${appointment.detailHeader.endDateTime}`}
+                  readOnly={true}
+                />
+              </Box>
+
+              {/* Tags */}
+              <Box sx={ParticipantAppointmentStyle.TextField}>
+                <Tags
+                  header="Tag"
+                  value={appointment.detailHeader.tags.map((t, index): Tag => {
+                    return { name: t, id: index };
+                  })}
+                  handleValueChange={() => {}}
+                  menuValue={[]}
+                  isErr={false}
+                  errMsg=""
+                  readOnly={true}
+                />
+                <FormHelperText>{"\u00A0"}</FormHelperText>
+              </Box>
+
+              {/* Description */}
+              <Box sx={ParticipantAppointmentStyle.TextField}>
+                <DescriptionTextField
+                  name="description"
+                  header="Description"
+                  placeholder="Enter Description Here"
+                  value={appointment.detailHeader.description}
+                  handleValueChange={() => {}}
+                  isErr={false}
+                  errMsg=""
+                  height={8}
+                  readOnly={true}
+                />
+              </Box>
+            </Stack>
+          </Card>
+          <Card sx={ParticipantAppointmentStyle.Card}>
+            <Stack spacing={3} alignItems="center" justifyContent="center">
+              {/* Image list */}
+              <Box sx={ParticipantAppointmentStyle.TextField}>
+                <DisplayImages header="Image" images={appointment.images} />
+              </Box>
```

Comment:
Same as in `host/[appointment_id].tsx` ka, add condition to check whether image exists or not.

Reply:
Solved